scripts/assemble_changelog.py

```diff
@@ -220,7 +220,8 @@ def add_categories_from_text(self, filename, line_offset,
 
             body_split = category.body.splitlines()
             for line_number, line in enumerate(body_split, 1):
-                if len(line) > MAX_LINE_LENGTH:
+                if not re.match('.*http[s]?://.*', line.decode('utf-8')) and \
```

Comment:
A very small amount of performance could possibly be eaked back by compiling the re above this function like the others. I am not sure how much performance this gives in modern python, but this is being used on every body line.

Reply:
Ah, sure. Will fix! ;)